assets/js/blocks/cart-checkout/cart-i2/inner-blocks/register-components.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { lazy } from '@wordpress/element';
+import { WC_BLOCKS_BUILD_URL } from '@woocommerce/block-settings';
+import { registerCheckoutBlock } from '@woocommerce/blocks-checkout';
+
+// Modify webpack publicPath at runtime based on location of WordPress Plugin.
+// eslint-disable-next-line no-undef,camelcase
+__webpack_public_path__ = WC_BLOCKS_BUILD_URL;
+
+/**
+ * Internal dependencies
+ */
+import filledCartMetadata from './filled-cart-block/block.json';
+import emptyCartMetadata from './empty-cart-block/block.json';
+import cartItemsMetadata from './cart-items-block/block.json';
+import cartExpressPaymentMetadata from './cart-express-payment-block/block.json';
+import cartLineItemsMetadata from './cart-line-items-block/block.json';
+import cartOrderSummaryMetadata from './cart-order-summary-block/block.json';
+import cartTotalsMetadata from './cart-totals-block/block.json';
+import cartProceedToCheckoutMetadata from './proceed-to-checkout-block/block.json';
+
+// @todo When forcing all blocks at once, they will append based on the order they are registered. Introduce formal sorting param.
```

Comment:
Note we have this TODO already in checkout.

Reply:
Wouldn't they merge? I dunno but I will delete